[pages] Update Jekyll guide instructions to use RUBY_VERSION (#7088)
cloudflare/cloudflare-docs: content/pages/framework-guides/deploy-a-jekyll-site.md

Jekyll is written in Ruby, meaning that you will need a functioning Ruby installation, like `rbenv`, to install Jekyll.

To install Ruby on your computer, follow the [`rbenv` installation instructions](https://github.com/rbenv/rbenv#installation) and select a recent version of Ruby by running the `rbenv` command in your terminal. The Ruby version you install will also be used to configure the Pages deployment for your application.

```sh

> $ rbenv install $RUBY_VERSION # For instance, 3.1.3

Reviewer: ```suggestion
$ rbenv install <RUBY_VERSION> # For example, 3.1.3
```

</div>

> Add an [environment variable](/pages/platform/build-configuration/#environment-variables) that matches the Ruby version that you are using locally. Set this as `RUBY_VERSION` on both your preview and production deployments. Below, `3.1.3` is used as an example:

Reviewer 2: ```suggestion
Add an [environment variable](/pages/platform/language-support-and-tools/) that matches the Ruby version that you are using locally. Set this as `RUBY_VERSION` on both your preview and production deployments. Below, `3.1.3` is used as an example:
```

> <div>
> 
> | Environment variable | Value          |
> | -------------------- | -------------- |
> | `RUBY_VERSION`       | `3.1.3`        |
> 
> </div>

Reviewer 2: ```suggestion
| Environment variable | Value          |
| -------------------- | -------------- |
| `RUBY_VERSION`       | `3.1.3`        |
```
